.buildkite/test-pipeline.yaml

```diff
@@ -333,12 +333,10 @@ steps:
   - vllm/model_executor/models/
   - tests/distributed/
   commands:
-  - # the following commands are for the first node, with ip 192.168.10.10 (ray environment already set up)
-    - VLLM_TEST_SAME_HOST=0 torchrun --nnodes 2 --nproc-per-node=2 --rdzv_backend=c10d --rdzv_endpoint=192.168.10.10 distributed/test_same_node.py | grep -q 'Same node test passed'
-    - VLLM_MULTI_NODE=1 pytest -v -s distributed/test_multi_node_assignment.py
-    - VLLM_MULTI_NODE=1 pytest -v -s distributed/test_pipeline_parallel.py
-  - # the following commands are for the second node, with ip 192.168.10.11 (ray environment already set up)
-    - VLLM_TEST_SAME_HOST=0 torchrun --nnodes 2 --nproc-per-node=2 --rdzv_backend=c10d --rdzv_endpoint=192.168.10.10 distributed/test_same_node.py | grep -q 'Same node test passed'
+  # the following commands are for the first node, with ip 192.168.10.10 (ray environment already set up)
+  - VLLM_TEST_SAME_HOST=0 torchrun --nnodes 2 --nproc-per-node=2 --rdzv_backend=c10d --rdzv_endpoint=192.168.10.10 distributed/test_same_node.py; VLLM_MULTI_NODE=1 pytest -v -s distributed/test_multi_node_assignment.py; VLLM_MULTI_NODE=1 pytest -v -s distributed/test_pipeline_parallel.py
+  # the following commands are for the second node, with ip 192.168.10.11 (ray environment already set up)
+  - VLLM_TEST_SAME_HOST=0 torchrun --nnodes 2 --nproc-per-node=2 --rdzv_backend=c10d --rdzv_endpoint=192.168.10.10 distributed/test_same_node.py
```

Comment:
Can we keep the `grep` command for these tests to ensure that the tests are actually run?